mappings/net/minecraft/client/render/model/json/ModelElement.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/class_785 net/minecraft/client/render/model/json/ModelElement
 	CLASS class_786 Deserializer
+		METHOD deserialize deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
```

Comment:
Please include this suggested change.
```suggestion
		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
```

Reply:
why? lol

Reply:
You are just not supposed to map this lol. In every other pr we don't include such duplicate names, especially for those screen class mappings

Reply:
How did this even get mapped? Enigma shouldn't generate names for these.

Reply:
This mapping relates to the `deserialize` method on all the deserializers:

```java
        public ModelElement method_3406/*deserialize*/(JsonElement json, Type type, JsonDeserializationContext context) throws JsonParseException {
            JsonObject jsonObject5 = json.getAsJsonObject();
            Vector3f vector3f6 = this.deserializeFrom(jsonObject5);
            Vector3f vector3f7 = this.deserializeTo(jsonObject5);
            ModelRotation modelRotation8 = this.deserializeRotation(jsonObject5);
            Map<Direction, ModelElementFace> map9 = this.deserializeFacesValidating(context, jsonObject5);
            if (jsonObject5.has("shade") && !JsonHelper.hasBoolean(jsonObject5, "shade")) {
                throw new JsonParseException("Expected shade to be a Boolean");
            }
            boolean boolean10 = JsonHelper.getBoolean(jsonObject5, "shade", true);
            return new ModelElement(vector3f6, vector3f7, map9, modelRotation8, boolean10);
        }
```

I think there's some bug in Enigma, though, because based on this signature this line in the mappings should look like this:

```
METHOD method_3406 deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Lnet/minecraft/class_785;